.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "go.testEnvVars": {
+        "ACG_CONFIG": "../../../tests/test.json"
+    },
+    "go.coverOnSave": true,
+    "go.coverageDecorator": {
+        "type": "gutter",
+        "coveredHighlightColor": "rgba(64,128,128,0.5)",
+        "uncoveredHighlightColor": "rgba(128,64,64,0.25)",
+        "coveredGutterStyle": "blockgreen",
+        "uncoveredGutterStyle": "blockred"
+    }
+
+}
```

Comment:
New line at end of file please

Reply:
sorry, I didn't know vscode is the one that does that to your files by default. fixed